beacon_node/lighthouse_network/src/rpc/codec/ssz_snappy.rs

```diff
@@ -482,9 +487,12 @@ fn handle_rpc_request<E: EthSpec>(
     spec: &ChainSpec,
 ) -> Result<Option<InboundRequest<E>>, RPCError> {
     match versioned_protocol {
-        SupportedProtocol::StatusV1 => Ok(Some(InboundRequest::Status(
-            StatusMessage::from_ssz_bytes(decoded_buffer)?,
-        ))),
+        SupportedProtocol::StatusV1 => {
+            metrics::inc_counter_vec_by(&metrics::TOTAL_RPC_REQUESTS_BYTES_RECEIVED, &["status"], decoded_buffer.len() as u64);
```

Comment:
if this is the right spot for this metric, I will add another ones here: `by_range`, `by_root`, etc.